presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -937,6 +937,30 @@ public void testCastNullToColumnTypes()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testCreateEmptyPartition()
+    {
+        String tableName = "empty_partition_table";
+        assertUpdate(format("" +
+                "CREATE TABLE %s " +
+                "WITH ( " +
+                " FORMAT = 'ORC', " +
+                "   partitioned_by = ARRAY['p_varchar'] " +
+                ") " +
+                "AS " +
```

Comment:
Creating an empty table should be sufficient. We should be able to create a new table and then add an empty partition. Is there a reason why another partition was needed, or something you wanted test?